debian/control

```diff
@@ -4,19 +4,23 @@ Section: misc
 Priority: optional
 Standards-Version: 3.9.8
 Build-Depends: cmake,
-               debhelper (>= 9),
-               libbrotli-dev,
-               libgif-dev,
-               libgmock-dev,
-               libgoogle-perftools-dev,
-               libgtest-dev,
-               libhwy-dev,
-               libjpeg-dev,
-               libopenexr-dev,
-               libpng-dev,
-               libwebp-dev,
-               pkg-config,
-Homepage: https://gitlab.com/wg1/jpeg-xl
+ debhelper (>= 9),
+ libbrotli-dev,
+ libgif-dev,
+ libgmock-dev,
+ libgoogle-perftools-dev,
+ libgtest-dev,
+ libhwy-dev,
+ libjpeg-dev,
+ libopenexr-dev,
+ libpng-dev,
+ libwebp-dev,
+ pkg-config,
+ doxygen,
```

Comment:
I don't think doxygen is needed to build the debian package. If I remember correctly we don't install the doxygen docs.

Reply:
When trying to build the man pages on launchpad, I got the following error: `-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)`

Reply:
It is not a fatal error, it just won't build the documentation pages which are not installed in debian anyway. The code does:
```
find_package(Doxygen)
if(DOXYGEN_FOUND)
...
```

We could make this QUIET or put a flag to disabled it to silence the warning in the debian build.

Reply:
I'll rebuild after uninstalling doxygen to see if the man pages build.  If they do, I'll update the commit to remove doxygen.